src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -1,11 +1,60 @@
-.editOnGithubLink {
-  margin-top: 64px;
+/*
+NOTE:
+Relies on `--sticky-bars-height` being set,
+this is currently being done in `BaseNewLayout`.
+See comment in src/layouts/base-new/base-new-layout.module.css
+for further details.
+*/
+
+.contentWrapper {
+  display: flex;
+  /* flex-grow ensures the content fills available
+  space in BaseNewLayout */
+  flex-grow: 1;
+
+  /* For a coherent visual effect, the top padding value
+  must be used for sticky positioning in addition to
+  padding the content area. So, we create a CSS var
+  to make it reusable and express this intent. */
+  --main-area-top-padding: 32px;
 }
 
-.footer {
-  /* the 50vh margin-top on footer ensures that all
-  jump-nav items have room to be jumped to */
-  margin-top: 50vh;
```

Comment:
This class didn't actually change, it is second from the bottom now. Weird diff!